tests/network/localnet/migration_stuntime/conftest.py

```
@@ -18,11 +18,29 @@
+from tests.network.localnet.migration_stuntime.libstuntime import CLIENT_VM_LABEL, SERVER_VM_LABEL, ContinuousPing
 
 LOGGER = logging.getLogger(__name__)
 
 
+@pytest.fixture(scope="class")
+def ip_family(request: pytest.FixtureRequest) -> int:
+    """IP family for stuntime measurement, activated by per-test parametrize."""
+    return request.param
+
+
+@pytest.fixture()
```

Ruff (0.15.13)

[warning] Use `@pytest.fixture` over `@pytest.fixture()`

Remove parentheses

(PT001)